pyproject.toml

```diff
@@ -60,6 +60,7 @@ select = [
 extend-select = [
     "C4",  # see: https://pypi.org/project/flake8-comprehensions
     "SIM",  # see: https://pypi.org/project/flake8-simplify
+#    "PT",  # see: https://pypi.org/project/flake8-pytest-style
```

Comment:
Is there a reason why this is commented?

Reply:
Because some more need to be resolved manually, so will check it later but first wanted to run the automatic fix and get your opinion...